packages/create-widget/src/templates.ts

```diff
@@ -42,8 +42,5 @@ export interface Template {
 
 export interface TemplateContext {
   project: string;
-  foundryUrl: string;
   osdkPackage: string;
-  widgetSetRid: string;
-  clientVersion: string;
```

Comment:
These aren't currently being used in any of the template handlebars but we could add them back in if we do use them later